### PR TITLE
CON-2908 Markdown Armstrong Number Exercise

### PR DESCRIPTION
**Pull request by zanninso**
[CON-2908](https://01talent.atlassian.net/browse/CON-2908)

[CON-2908]: https://01talent.atlassian.net/browse/CON-2908?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MSilva95 — CHANGES_REQUESTED** [review]
Format, and it's good to go.
Again, I'm not a fan of all those comments in the usage, for example why would we leave comments like " // Expected output: true " if we actually have the expected output below?

File: subjects/java/checkpoints/armstrong-number/ExerciseRunner.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+        // Test case 2
+        int number2 = 123;
+        System.out.println("Is " + number2 + " an Armstrong number? " + armstrongNumber.isArmstrong(number2)); // Expected output: false
+
+        // Test case 3
+        int number3 = 9474;
+        System.out.println("Is " + number3 + " an Armstrong number? " + armstrongNumber.isArmstrong(number3)); // Expected output: true
+    }
+}
```
[the remark this answers is not in the document]
Comment (zanninso) [reply subjects/java/checkpoints/armstrong-number/ExerciseRunner.java:17]: what should i format here ?

[the remark this answers is not in the document]
Comment (zanninso) [reply subjects/java/checkpoints/armstrong-number/ExerciseRunner.java:17]: this is a java code it's formatted already

[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/java/checkpoints/armstrong-number/ExerciseRunner.java:17]: image

**Review by MSilva95 — APPROVED** [review]
LGTM!
You can close the conversation if you feel no need to format the runner.java

**Pull request merged**